guide.md

```diff
@@ -96,3 +132,37 @@ For Shibboleth the loginInitiatorURL serves both purposes but in general you nee
 The *discoveryRequest* function is called with a single argument containing a JS object representing the chosen IdP - parameters include entity ID, icon, title etc. Normally this function will be used to initialize a SAML authentication request to the identity provider identified by *entity.entityID*.
 
 For Shibboleth this is all handled by providing the single *loginInitiatorURL* parameter as in the example above.
+
+---
+
+# Implementing an "Advanced" Integration
+
+![Illustration of the advanced integration](/assets/img/SA-advanced.png)
+
+For the "Advanced" flavor, you will implement:
+
+* Download and edit the [Seamless Access Discovery service](https://github.com/TheIdentitySelector/thiss-ds-js) software - _you will use this set of clients to configure both persistence and discovery services for your custom service_
+* Use the Seamless Access API - _Use these clients to interact with the persistence service when using your own discovery service._
+
+The "Advanced" integration is significantly more complicated than the other implementations, and should only be used if you are unable to achieve your discovery and/or UX experiences otherwise. 
+
+## The Seamless Access Discovery Service Software
+
+Seamless Access is registered as a [package](https://www.npmjs.com/package/@theidentityselector/thiss-ds) on [npm](https://www.npmjs.com/). You can install the latest version of Seamless Access with the npm CLI command
+
+``` bash
+> npm install [--save] @theidentityselector/thiss-ds
+```
+
+Once downloaded, you can reference the (edited) software in your HTML.
+
+``` html
+<head>
+<!-- Include the downloaded Seamless Access Discovery Client API -->
+<script src="/thiss-ds.js"></script>
+</head>
+```
+
+## Use the Seamless Access API
+
+The documentation for the API can be found in the [software README file](https://github.com/TheIdentitySelector/thiss-ds-js/blob/master/README.md).
```

Comment:
I was trying to reconcile the Browser Local Storage JSON schema shown in the readme file with what I can see in the local store on my machine when interacting with the button. There were some fields present in one and not the other, and in some cases (like entity ID), the naming isn't quite the same (for example, `entity_id` vs `entityID`). Is this documentation that needs to be updated?